meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx8mmevk/lmp.cfg

```diff
@@ -13,7 +20,7 @@ CONFIG_SPL_LOAD_FIT=y
 CONFIG_OF_LIBFDT_OVERLAY=y
 CONFIG_FIT=y
 CONFIG_FIT_SIGNATURE=y
-CONFIG_FIT_SIGNATURE_STRICT=y
```

Comment:
Same here.  We need this to stay in place.

Reply:
will fix